pkg/configschema/common_test.go

```diff
@@ -25,30 +21,31 @@ import (
 	"go.opentelemetry.io/collector/config/configtls"
 )
 
-type testPerson struct {
-	Name string
-}
+const DefaultModule = "github.com/open-telemetry/opentelemetry-collector-contrib"
 
 // testStruct comment
 type testStruct struct {
```

Comment:
I think the reordering here makes the test a bit confusing, could you get `One`, `Two`, `Three` and `Four` to be consecutive again?